py/desispec/sky.py

```diff
@@ -1010,8 +1013,9 @@ def __init__(self, wave, flux, ivar, mask, header=None, nrej=0, stat_ivar=None):
         self.header = header
         self.nrej = nrej
         self.stat_ivar = stat_ivar
+        self.throughput_corrections = throughput_corrections
 
-def subtract_sky(frame, skymodel, throughput_correction = False) :
+def subtract_sky(frame, skymodel, throughput_correction = True) :
```

Comment:
Stylistic: it is confusing to me reading the code that "throughput_correction" is a boolean about whether to apply the throughtput correction, while "skymodel.throughput_corrections" is the correction itself.  Consider making these names differ by more than "s", e.g. with "apply_throughput_correction" or "apply_thrucorr" for the boolean.

Reply:
That also predates my changes as well, but it's a great idea. I'll go with apply_throughput_correction